theano/gpuarray/opt.py

```diff
@@ -1298,7 +1291,7 @@ def local_inplace_sparseblockouter(node):
 
 
 # This deals with any abstract convs that have a transfer somewhere
-@register_opt('fast_compile', 'conv_dnn')
```

Comment:
@lamblin  asked me to remove. I'm not sure of these tags..
https://github.com/Theano/Theano/pull/4570#discussion_r69383088